tachyon/math/finite_fields/generator/ext_field_generator/build_defs.bzl

```
@@ -1,7 +1,7 @@
 load("//bazel:tachyon_cc.bzl", "tachyon_cc_library")
 
 def _generate_ext_field_impl(ctx):
-    fq_hdr_tpl_path = ctx.expand_location("$(location @kroma_network_tachyon//tachyon/math/finite_fields/generator/ext_field_generator:fq.h.tpl)", [ctx.attr.fq_hdr_tpl_path])
+    fq_hdr_tpl_path = ctx.expand_location("$(location @kroma_network_tachyon//tachyon/math/finite_fields/generator/ext_field_generator:fp.h.tpl)", [ctx.attr.fq_hdr_tpl_path])
```

+ fix for all instances of `fq_hdr_tpl_path`
```suggestion
    fp_hdr_tpl_path = ctx.expand_location("$(location @kroma_network_tachyon//tachyon/math/finite_fields/generator/ext_field_generator:fp.h.tpl)", [ctx.attr.fp_hdr_tpl_path])